### PR TITLE
docs: add Japanese README

### PR DESCRIPTION
**Pull request by eltociear**
## Motivation

I created Japanese translated README.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: README-ja-JP.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+>>> model = AutoModelForCausalLM.from_pretrained("internlm/internlm-chat-7b", trust_remote_code=True).cuda()
+>>> model = model.eval()
+>>> response, history = model.chat(tokenizer, "hello", history=[])
+>>> print(response)
+こんにちは！ 今日はどのようなご用件でしょうか？
+>>> response, history = model.chat(tokenizer, "please provide three suggestions about time management", history=history)
+>>> print(response)
+もちろん、効果的な時間管理のための3つのヒントはここにあります:
+
+1. 重要度と緊急度に基づいてタスクの優先順位をつける: すべてのタスクをリストアップし、"重要かつ緊急"、"重要だが緊急ではない"、"重要ではないが緊急"に分類する。最初のカテゴリーのタスクを完了させることに集中してから、他のカテゴリーに移る。
```
Comment (ZwwWayne) [inline README-ja-JP.md:93]: Hi @eltociear ，
Just want to make sure that are these contents produced by the InternLM-Chat or simply translated from the origin documents?

**Comment by ZwwWayne** [comment]
Hi @eltociear ,
Thanks for your kind contribution.
I have translated the Japanese contents back to Chinese via Bing and they looks good to me.
This PR can be merged once we got confirmation from you that the contents are indeed produced by the model

File: README-ja-JP.md (hunk 1 above)
Comment (eltociear) [reply README-ja-JP.md:93]: @ZwwWayne Sorry, this is simply my translation of the original text.

**Comment by eltociear** [comment]
@ZwwWayne Yes, I will replace this one.

**Comment by eltociear** [comment]
@ZwwWayne I have replaced the target string with the string of the output content.

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**